pyproject.toml

```diff
@@ -170,6 +170,18 @@ src_paths = ["doctr", "tests", "scripts", "references", "demo", "docs", "api"]
 skip_glob = "**/__init__.py"
 known_third_party = ["tensorflow", "torch", "torchvision", "wandb", "fastprogress"]
 
+[tool.ruff]
+ignore = ["E402", "F403", "E731"]
+exclude = [".github", ".git", "venv*", "docs", "build"]
```

Comment:
same here (if there are python script in .github or docs, it might be nice to have those checked by ruff?)

Reply:
Sure i have no use Case in mind currently but why not :)